examples/moonolith/ex1p.cpp

```diff
@@ -0,0 +1,245 @@
+//                                MFEM Example 1 - Parallel Version
+//                              Moonolith Modification
+//
+// Compile with: make ex1p
+//
+// Moonolith sample runs:
+//               mpirun -np 4 ex1p
+//               mpirun -np 4 ex1p --source_refinements 1 --dest_refinements 2
+//               mpirun -np 4 ex1p --source_refinements 1 --dest_refinements 2 --use_vector_fe
```

Comment:
We are aware of the issue with RT,  I described it as an experimental feature in the doc. The problem appears only in parallel runs... maybe I am missing some permutation since the sum of the operator seems to be consistent with serial runs. Any help in this matter would be appretiated.